validphys2/src/validphys/config.py

```diff
@@ -1261,7 +1262,7 @@ def parse_filter_rules(self, filter_rules: (list, type(None))):
         """A list of filter rules. See https://docs.nnpdf.science/vp/filters.html
         for details on the syntax"""
         log.warning("Overwriting filter rules")
-        return filter_rules
+        return tuple(frozendict(rule) for rule in filter_rules) if filter_rules else None
```

Comment:
Please  change the docstr from list to tuple. You might want to mention ad well that each rule is given as an inmutable dictionary.